### PR TITLE
NO-ISSUE: Organizing the repository to share common resources 

### PR DESCRIPTION
**Pull request by danmanor**
Organizing the repository in order to:

1. Share current common resources.
2. Share future common resources (I am going to add shared `usages` index)

TODO: change repository name

### CODE REVIEW

File: tests/prowjobsscraper/test_step.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 import pkg_resources
 from google.cloud import exceptions
 
-from prowjobsscraper import step
-from prowjobsscraper.prowjob import ProwJobs
+from projects.prowjobsscraper import step
```
Comment (osherdp) [inline tests/prowjobsscraper/test_step.py:8]: I think it's kind of a convention that repository name and the root package have the same name, ``projects`` kind of negates this concept (I would expect ``prowjobsscraper`` to be the root, or maybe we can rename the repo name to something more generic)

Comment (danmanor) [reply tests/prowjobsscraper/test_step.py:8]: Yes I agree, I will change the repository name as well, do you have an idea ?

Comment (osherdp) [reply tests/prowjobsscraper/test_step.py:8]: The best I can think right now is ``prow-jobs-reporter``

Comment (adriengentil) [reply tests/prowjobsscraper/test_step.py:8]: I named it `prow-insights` in app-interface, we could align with this name?

**Pull request closed without merging**